docs/getting-started/auth-self-defence-checks.adoc

```diff
@@ -154,4 +164,32 @@ Map<String, SecurityCheckResult> results = securityService.getCheckExecutor().ad
 
 // adding self defence checks and executing these checks asynchronously
 Map<String, Future<SecurityCheckResult> results = securityService.getAsyncCheckExecutor().addCheck(new customSecurityCheck()).addCheck(SecurityCheckType.<check_type>).addCheck(SecurityCheckType.<check_type>).execute();
-----
\ No newline at end of file
+----
+
+=== Reporting Self-defence checks results via the Metrics service
+
+In order to report the results of Self-defence utilize the link:./metrics[Metrics] service in conjunction with the Security service.
```

Comment:
Maybe refactor this to "To report the results of self defence checks, utilize the link:./metrics[Metrics] service in conjunction with the link:javdoc[`SecurityService`]" ??